pilot/pkg/bootstrap/istio_ca.go

```diff
@@ -479,11 +493,13 @@ func (s *Server) createIstioCA(opts *caOptions) (*ca.IstioCA, error) {
 	return istioCA, nil
 }
 
-func (s *Server) createSelfSignedCACertificateOptions(fileBundle *ca.SigningCAFileBundle, opts *caOptions) (*ca.IstioCAOptions, error) {
+// loadCAFromSecrets will initialize the Istio CA using istio-ca-secrets or cacerts.
+// This is only called if mounted files
+func (s *Server) loadCAFromSecrets(fileBundle *ca.SigningCAFileBundle, opts *caOptions) (*ca.IstioCAOptions, error) {
```

Comment:
```suggestion
func (s *Server) newSelfSignedIstioCAOptions(fileBundle *ca.SigningCAFileBundle, opts *caOptions) (*ca.IstioCAOptions, error) {
```

Reply:
The problem is that it seems we may have casecrets get 'self signed' - in which case it will be mounted. 

How about loadSelfSignedFromSecrets ?

Reply:
`loadOrCreateCAFromSecret`?

Reply:
Sounds good - but can we do this in another PR, want to get this in so it gets more time ( and I have few other changes waiting for this one)